tests/UI/campaigns/functional/FO/06_search/01_searchProduct.js

```diff
@@ -1,55 +0,0 @@
-require('module-alias/register');
```

Comment:
I think you should keep this test.

Reply:
I don't know why the merge remove this file :(

Reply:
Thank you